appengine/ndb/async/app_async.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.appengine.api import users
+from google.appengine.ext import ndb
+import jinja2
+import webapp2
+
+
+class Account(ndb.Model):
+    view_counter = ndb.IntegerProperty()
+
+
+class MyRequestHandler(webapp2.RequestHandler):
+    def get(self):
+        acct = Account.get_by_id(users.get_current_user().user_id())
+        acct.view_counter += 1
+        future = acct.put_async()
+
+        # ...read something else from Datastore...
+        template_values = {}
+
+        self.response.out.write(template.render(template_values))
+        future.get_result()
+
+app = webapp2.WSGIApplication([('/', MyRequestHandler)])
+
+# Set up a generic template just for demonstration
```

Comment:
Don't render the template? Just write out. You're already handwaving a lot in this sample.